frontend/src/container/Login.jsx

```diff
@@ -268,6 +268,20 @@ class Login extends React.Component {
 
   handleUserConnection = async () => {
     const { props } = this
+
+    if (window.Notification) {
+      try {
+        if (Notification.permission !== 'denied') {
+          const permission = Notification.requestPermission()
+          if (permission === 'granted') {
+            new Notification('notification granted') // eslint-disable-line no-new
```

Comment:
Should 'notification granted' be translated? I don't know if this `if` is not a forgotten debug

Reply:
Yes, we probably don't need to create a notification here, how about I just delete this condition?